book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Please use a getter and setter method for rentals also. In other words, kindly add it to `` attr_accessor``